packages/react/counter/src/__test__/mswHandlers.ts

```diff
@@ -0,0 +1,19 @@
+import { graphql } from 'msw'
+
+import { server } from '@/testing/mocks/server'
+
+import { counter } from './mockData'
+
+const handlers = [
+  graphql.query('Counter', (req, res, ctx) => {
+    return res(
+      ctx.data({
+        counter
+      })
+    )
```

Comment:
Nice 👍 Since we are here, how about expanding this to show

- [x] Returning a value.
- [ ] Returning `null` due to lack of data (this will happen for AVERAGE, MIN, MAX, etc.).
- [ ] Returning `null` due to error (this could happen due to timeout, permissions, etc.).

And going ahead and getting the tests for the second two into the &lt;Counter&gt; component's tests? I'm assuming they will just pass but, if they don't we can skip them and address in a followup.

Reply:
Yes! I just added tests for those